mne/io/eeglab/eeglab.py

```diff
@@ -631,13 +631,14 @@ def __init__(
                     event_type = "/".join([str(et) for et in ep.eventtype])
                     event_name.append(event_type)
                     # store latency of only first event
```

Comment:
If this comment is right: https://github.com/jackz314/eeglabio/blob/f496ba45b5db716360e05c24efd89339f7bd2434/eeglabio/epochs.py#L101

Then MATLAB stores sample number from 1-onward, so we need to account for that when reading event latencies (in sample space) back in. That sounds right to me since MATLAB is a 1-based indexing language, but I would appreciate it if someone else with more MATLAB experience can confirm.